docs/administering/active-directory.md

```diff
@@ -0,0 +1,304 @@
+This document explains how to integrate with Microsoft Active Directory (also known as AD).  This
+allows all users, or specific users within your Windows domain, to log into a Sandstorm server. The
+integration requires [Sandstorm for Work.](for-work.md)
+
+## AD & Sandstorm: Overview
+
+Sandstorm can rely on AD to authenticate users.
+
+Active Directory exists as a cloud product called Windows Azure as well as a software product called
```

Comment:
"exists as a" -> "exists as part of a"

Otherwise I feel like this is saying that Windows Azure and Windows Server are versions of Active Directory, rather than bundles that include versions of Active Directory...